app/templates/results.html

```diff
@@ -85,8 +84,21 @@ <h4>License:</h4>
         {% endif %}
         
       </div>
-      <button class="show_more_button" >Show more</button>
-      <button class="show_less_button" >Show less</button>
+      <!-- only show the buttons if javascript is enabled (nojs not defined) -->
+      {% if nojs is not defined %}
+        <button class="show_more_button" >Show more</button>
+        <button class="show_less_button" >Show less</button>
+      {% endif %}
+      
+
+
+      <!--if javascript is not enabled,default in showing the result in full-->
+       <noscript>
+        <style type="text/css">
```

Comment:
You can do this in the scss files with something like `.results__container--no-js .result--full { display: inline-block; }` and that'll be faster, too.